mlflow/utils/proto_json_utils.py

```diff
@@ -475,7 +477,8 @@ def get_jsonable_input(name, data):
         raise MlflowException(f"Incompatible input type:{type(data)} for input {name}.")
 
 
-def dump_input_data(data, inputs_key="inputs"):
+@experimental
```

Comment:
Nit: The `params` field should be marked experimental in the docstring, but the method as a whole should not be marked `experimental`.